crates/uv-configuration/src/target_triple.rs

```diff
@@ -27,7 +27,7 @@ pub enum TargetTriple {
     #[cfg_attr(feature = "clap", value(name = "x86_64-pc-windows-msvc"))]
     X8664PcWindowsMsvc,
 
-    /// An x86 Linux target.
+    /// An x86 Linux target. Equivalent to `x86_64-manylinux_2_17`.
     #[cfg_attr(feature = "clap", value(name = "x86_64-unknown-linux-gnu"))]
     X8664UnknownLinuxGnu,
```

Comment:
We could consider removing these and _only_ supporting the manylinux versions? And change `x86_64-unknown-linux-musl` to `x86_64-musllinx_1_2`?